File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1020_basico.js
import { question } from "readline-sync";

function main() {
    const idade = get_number();

    let valor = Calcular_Idade(idade);

    print(`${valor[0]} ano (s)`);
    print(`${valor[1]} mes (es)`);
    print(`${valor[2]} dia (s)`);

}

function get_number() {
    return Number(question());
}

function print(texto) {
    return console.log(texto);
}

function Calcular_Idade(num) {
    let ano = Math.floor(num / 365);
    let mes = Math.floor((num % 365) / 30);
    let dia = Math.floor((num % 365) % 30);

    return [ano, mes, dia]
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1020_basico.py
import math

td = int(input())
a = math.floor(td / 365)
m = math.floor((td % 365) / 30)
d = math.floor((td % 365) % 30)

print(f"{a} ano (s)")
print(f"{m} mes (s)")
print(f"{d} dia (s)")